internal/uvm/scsi.go

```diff
@@ -91,27 +120,29 @@ func (sm *SCSIMount) RefCount() uint32 {
 
 func (sm *SCSIMount) logFormat() logrus.Fields {
 	return logrus.Fields{
-		"HostPath":        sm.HostPath,
-		"UVMPath":         sm.UVMPath,
-		"isLayer":         sm.isLayer,
-		"refCount":        sm.refCount,
-		"Controller":      sm.Controller,
-		"LUN":             sm.LUN,
-		"SerialVersionID": sm.serialVersionID,
+		"HostPath":                  sm.HostPath,
+		"UVMPath":                   sm.UVMPath,
+		"isLayer":                   sm.isLayer,
+		"refCount":                  sm.refCount,
+		"Controller":                sm.Controller,
+		"LUN":                       sm.LUN,
+		"SerialVersionID":           sm.serialVersionID,
+		"ExtensibleVirtualDiskType": sm.extensibleVirtualDiskType,
 	}
 }
 
-func newSCSIMount(uvm *UtilityVM, hostPath, uvmPath, attachmentType string, refCount uint32, controller int, lun int32, readOnly bool) *SCSIMount {
+func newSCSIMount(uvm *UtilityVM, hostPath, uvmPath, attachmentType, evdType string, refCount uint32, controller int, lun int32, readOnly bool) *SCSIMount {
 	return &SCSIMount{
-		vm:              uvm,
-		HostPath:        hostPath,
-		UVMPath:         uvmPath,
-		refCount:        refCount,
-		Controller:      controller,
-		LUN:             int32(lun),
-		readOnly:        readOnly,
-		attachmentType:  attachmentType,
-		serialVersionID: scsiCurrentSerialVersionID,
+		vm:                        uvm,
+		HostPath:                  hostPath,
+		UVMPath:                   uvmPath,
+		refCount:                  refCount,
+		Controller:                controller,
+		LUN:                       int32(lun),
+		readOnly:                  readOnly,
+		attachmentType:            attachmentType,
+		serialVersionID:           scsiCurrentSerialVersionID,
```

Comment:
For clarity I think we should keep `serialVersionID` as either the first or last element of this list.